docs/user/installation.rst

```diff
@@ -93,7 +93,10 @@ pip:
 
 .. code-block:: bash
 
+    # bash
```

Comment:
`pip install 'garage[mujoco,dm_control]'` works in both bash and zsh, so it's preferable to just use that

Reply:
same comment for all of the below